test/ex_doc/formatter/html_test.exs

```diff
@@ -102,4 +124,34 @@ defmodule ExDoc.Formatter.HTMLTest do
     assert HTML.pretty_codeblocks(iex_detected_without_class) ==
           "<pre><code class=\"iex elixir\">iex&gt; max(4, 5)"
   end
+
+  test "run generates the redirect index.html" do
+    config = doc_config([project: "Elixir", version: "1.0.1", output: "#{output_dir}/redirect", main: "RandomError", ])
+    generate_docs(config)
+
+    assert File.regular?("#{output_dir}/redirect/RandomError.html")
+    content = File.read!("#{output_dir}/redirect/index.html")
+    assert content =~ ~r{<meta http-equiv="refresh" content="0; url=RandomError.html"\s*/>}
+  end
+
+  test "run generates index.html and normalized options" do
+    for dir <- ["test/tmp/doc", "test/tmp/doc/", "test/tmp/doc//"] do
```

Comment:
this is the only suggestion that I didn't implement. I want your opinion first,
While it is double checking something we are checking in the ex_doc_test.exs, the inclusion of this  test was the one that made me realize that things in this tests were not running through ExDoc module, and that why the overhaul.
So that's why it makes me feel a bit uneasy to not to have something like this, at least to let us rest assure that we are generating the docs the right way (ie. not through HTML.run)
What do you think?